### PR TITLE
Converted 7 docs to .adoc

### PR DESCRIPTION
**Pull request by Preeticp**
## What is the purpose of this change? What does it change?

- This PR converts 7 docs to .adoc format. This takes care off all the docs in the docs folder to enable change in the script for the website.
- It also reviews the Examples file.
- The other 6 docs as discussed with @kadel  and @cdrage , are to be maintained by the developers, are not the first priority for GA and can be revisited later. They have been converted to adoc, and have a very basic sanity check done.

### CODE REVIEW

**Comment by Preeticp** [comment]
@cdrage I think you can go ahead and merge this, so that you can change the script for the website.

File: docs/cli-reference.md [hunk 1]
```diff
@@ -7,12 +7,12 @@
   git clone https://github.com/openshift/nodejs-ex && cd nodejs-ex
   odo create nodejs
   odo push
-  
+
```
Comment (cdrage) [inline docs/cli-reference.md]: cli-reference.md is automatically generated. Can you revert your changes to `cli-reference.md` ?

**Pull request merged**